src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/ExtendedQueryTagError.cs

```diff
@@ -33,7 +33,7 @@ public class ExtendedQueryTagError
 
         public string SopInstanceUid { get; }
 
-        public DateTime UtcTimestamp { get; }
+        public DateTime UtcCreatedTime { get; }
```

Comment:
I think based on our discussion we're not going to include `Utc` in the name, right? Because we don't do it elsewhere?